modules/metering-prometheus-connection.adoc

```diff
@@ -7,7 +7,7 @@
 
 When you install metering on {product-title}, Prometheus is available at https://prometheus-k8s.openshift-monitoring.svc:9091/.
 
-To secure the connection to Prometheus, the default metering installation uses the OpenShift certificate authority. If your Prometheus instance uses a different CA, the CA can be injected through a `ConfigMap`. See the example below.
+To secure the connection to Prometheus, the default metering installation uses the OpenShift certificate authority. If your Prometheus instance uses a different CA, the CA can be injected through a ConfigMap. See the example below.
```

Comment:
Never "OpenShift" by itself. s/OpenShift/{product-title}
You can use OpenShift as part of a specific component name if you must.

Reply:
(never "below")